### PR TITLE
Adding Code of Conduct 🚀

### PR DESCRIPTION
**Pull request by annakrystalli**
I've adapted the Galaxy Code of Conduct to our needs. I've also set up an Incident Report form and a private googlegroup for communication with  the Code of Conduct committee

### CODE REVIEW

**Comment by annakrystalli** [comment]
I've invited specific reviews but any comments and feedback from the team welcome!

**Comment by flor14** [comment]
For me it is perfect. I can not think on how to improve it.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+ReproHack Project Code of Conduct
+==============================
+
+This code of conduct outlines our expectations for participants within the
+ReproHack community, as well as steps to reporting unacceptable behavior. We are
+committed to providing a welcoming and inspiring community for all and expect
+our code of conduct to be honored. Anyone who violates this code of conduct may
+be banned from the community.
```
Comment (DanielaGawehns) [inline CODE_OF_CONDUCT.md]: Maybe online and offline??

Comment (annakrystalli) [reply CODE_OF_CONDUCT.md]: Good call. In fact I'll adapt something in the Carpentries CoC:

> Everyone participating in The Carpentries activities is required to conform to the Code of Conduct. This Code of Conduct applies to all spaces managed by The Carpentries including, but not limited to, workshops, email lists, and online forums such as GitHub, Slack and Twitter. Workshop hosts are expected to assist with the enforcement of the Code of Conduct.

Comment (DanielaGawehns) [reply CODE_OF_CONDUCT.md]: Yes, great. That should cover most online fora/spaces

Comment (annakrystalli) [reply CODE_OF_CONDUCT.md]: Done!

**Pull request merged**